.github/workflows/dependabot-batcher.yml

```diff
@@ -0,0 +1,38 @@
+# Dependabot Batcher v1.0.1
+
+name: 'Dependabot Batcher'
+
+# Controls when the workflow will run
+on:
+  push:
+    branches: [ "r/12.x", "develop" ]
+  pull_request:
+    branches: [ "r/12.x", "develop" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  dependabot-batcher:
+    name: 'Combine dependabot PRs'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repo'
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.DEPENDABOT_BATCHER }}
```

Comment:
If you react to pushes, you don't need this.

Reply:
Now reading from the developers of this action. They run in a scheduled way instead of for every push.

```
on:
  schedule:
    - cron:  '0 9 * * 1,3'
```

I think that is the reason to use a token. I will try with main repo on how behaves without this `token`

Reply:
Works well without the `checkout repo` step.